modules/pel/peltool/peltool.py

```diff
@@ -211,29 +211,43 @@ def severityBasedExclusion(uh: UserHeader, config: Config) -> bool:
             return False
     return True
 
-def excludePEL(uh: UserHeader, config: Config) -> bool:
+def considerPEL(uh: UserHeader, config: Config) -> bool:
     """
     Evaluates whether a PEL meets criteria based on specified conditions.
-    Returns True if the entry should be Excluded, False otherwise.
+    Returns True if the entry should be Included, False otherwise.
     """
-    if config.critSysTerm and uh.eventSeverity != SeverityValues.critSysTermSeverity.value:
+    isHidden = uh.actionFlags & ActionFlagsValues.hiddenActionFlag.value
```

Comment:
I think you planned to push the utility API (`isHidden()`)definition into the `uh` module. I would prefer it to be in this PR, but you could include it in your next PR.

Reply:
Created [PR59](https://github.com/ibm-openbmc/openpower-pel-parsers/pull/59) for above requested changes.